### PR TITLE
feat: unify results preview carousels styles

### PR DESCRIPTION
**Pull request by ajperezbau**
To keep in mind:

- Finally, we will always render the query as a button, independently of the number of results.
- In mobile view, the results of all the carousels have the same width. It is `x-w-[calc(38vw-16px)]` to always show two results and a half, regardless of the viewport's width. This is approved by @cris-crash.

EX-7325

### CODE REVIEW

File: src/components/pre-search/custom-query-preview.vue [hunk 1]
```diff
@@ -7,53 +7,30 @@
...
-          >
-            <li v-for="result in results" :key="result.id">
-              <Result :result="result" class="x-query-preview__item" />
-            </li>
-          </ul>
-          <template #sliding-panel-right-button>
-            <ChevronRightIcon class="x-icon-lg" />
-          </template>
-        </SlidingPanel>
+          <ItemsList :items="results" class="x-flex x-gap-16 x-pt-16 max-desktop:x-px-16">
```
Comment (annacv) [inline src/components/pre-search/custom-query-preview.vue:28]:
```suggestion
          <ItemsList :items="results" class="x-flex x-gap-16 x-pt-4 max-desktop:x-pt-0 max-desktop:x-px-16">
```
Distance between `BaseEventButton` it's less in designs, 16px & 10px mobile. As the button has its own white space inside, I think the padding defined should be a bit less.

**Review by annacv — CHANGES_REQUESTED** [review]
Hi! I've seen some little things : 
- Less distance between `BaseEventButton` and `ItemsList`:  as the button has its own internal white space the padding defined should be less.
- Product Name is in uppercase
- Mobile: There is no Product Description
- Mobile: In `NextQueryPreview`, the distance between the text and the button should be the same when there are 2 lines of text + button, check it for 375px screens as below:
Captura de pantalla 2023-02-20 a les 9 12 04

**Review by mnavarroespinosa — CHANGES_REQUESTED** [review]
good job!

File: src/components/my-history/custom-my-history.vue [hunk 2]
```diff
@@ -79,13 +79,12 @@
...
                   :suggestion="suggestion"
+                  suggestionClass="x-suggestion"
                 >
                   <div
                     class="x-list x-list--horizontal x-list--gap-05 x-padding--bottom-02 x-padding--top-02"
                   >
-                    <HistoryIcon
-                      :class="{ 'x-icon-lg': $x.device === 'mobile' }"
-                    />
+                    <HistoryIcon :class="{ 'x-icon-lg': $x.device === 'mobile' }" />
```
Comment (tiborux) [inline src/components/my-history/custom-my-history.vue:87]:
```suggestion
                    <HistoryIcon class="max-desktop:x-icon-lg" />
```

**Comment by herrardo** [comment]
This seems great!

**Review by annacv — APPROVED** [review]

**Review by mnavarroespinosa — APPROVED** [review]
Looks good to me, good job!

**Pull request merged**